packages/ui/src/i18n/dictionaries/index.ts

```diff
@@ -1,7 +1,15 @@
-export { deDict } from './de';
-export { enDict } from './en';
-export { esDict } from './es';
-export { frDict } from './fr';
-export { itDict } from './it';
-export { jaDict } from './ja';
-export { zhDict } from './zh';
+import * as authenticatorDict from './authenticator';
+
+//merge all the new module translations in respective locale constants
+export const deDict = { ...authenticatorDict.deDict };
```

Comment:
Just adding this for my own understanding: this change is enabling this in the future:

```suggestion
export const deDict = {
  ...authenticatorDict.deDict,
  ...someOtherDict.deDict
};
```